libpromises/dbm_api.c

```diff
@@ -66,6 +76,7 @@ struct DBCursor_
 static pthread_mutex_t db_handles_lock = PTHREAD_ERRORCHECK_MUTEX_INITIALIZER_NP; /* GLOBAL_T */
 
 static DBHandle db_handles[dbid_max] = { { 0 } }; /* GLOBAL_X */
+static DynamicDBHandles *db_dynamic_handles;
```

Comment:
Why? Why do we need a dynamic amount of databases? Haven't we had enough trouble with only a static number of those? ;-)